core/src/main/kotlin/in/specmatic/stub/api.kt

```diff
@@ -70,6 +71,13 @@ fun createStubFromContracts(
     host: String = "localhost",
     port: Int = 9000
 ): ContractStub {
+    val validContractPaths = contractPaths.filter { path ->
```

Comment:
The logic for filtering is duplicated 3 times. Can you pull it out into a common function?